app/models/setting.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+class Setting < ApplicationRecord
```

Comment:
@Letiste I step in here, we need full reliable control over the id of the next invoice we want to generate, for the invoices to be legaly compliant, in case a part of the db is cleared, or we need to purposly skip ids to account for manual invoices, or just the fact that we have to start the id at 7000-ish to ensure continuation from re2o. 
Therefore the generated id is not flexible enough.